src/main.js

```diff
@@ -1300,14 +1300,8 @@ function appendArtworkCardsFromShardsAndItems(shards, items, proxy, type, direct
                         title: entry.title,
                         author: entry.author,
                         thumbnail: thumbnail,
-                        desktop_id: proxy.desktopId,
-                        bus_name: proxy.busName,
-                        knowledge_search_object_path: proxy.knowledgeSearchObjectPath,
-                        knowledge_app_id: proxy.knowledgeAppId,
-                        uri: entry.ekn_id,
```

Comment:
It absolutely should eventually; feel free to open a follow up story or take this up and change to make it so.